docs/source/config_file.rst

```diff
@@ -439,6 +439,11 @@ These options may only be set in the global section (``[mypy]``).
 ``show_absolute_path`` (bool, default False)
     Show absolute paths to files.
 
+``ignore_error_codes`` (comma-separated list of strings)
+    Ignore errors with these error codes in given files or directories.
```

Comment:
The part "in given files or directories" seems redundant. It's not included in other descriptions, and so it raises the question why this option applies to given files or directories, while other options don't apply to given files or directories?